widget/package.json

```diff
@@ -27,7 +29,11 @@
     "npm-run-all": "^4.1.5",
     "parcel": "^2.10.0",
     "rimraf": "^5.0.5",
-    "typescript": "^5.2.2"
+    "typescript": "^5.2.2",
+    "@tsconfig/svelte": "^5.0.2",
```

Comment:
Please remove `@tsconfig/recommended` as a dependency, since it's no longer used.